.github/workflows/windows_test_comment_trigger.yml

```diff
@@ -0,0 +1,87 @@
+name: Test windows on the comment
+
+on:
+  issue_comment:
```

Comment:
Reading through docs it seems like issue_comment is probably the only option, it's really strange it doesn't have PR sha. At the same time all pull_request* actions have PR sha. Have you actually tried this one for example? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review
I'm not sure what this statement really means: 
```
A pull request review is a group of pull request review comments in addition to a body comment and a state
```

Reply:
This event triggers the workflow when a review comment is added or edited. Review comment != pull request comment.